src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFileUploadsSection.tsx

```diff
@@ -81,14 +84,37 @@ const ReleaseFileUploadsSection = ({ publicationId, releaseId }: Props) => {
   };
 
   const handleSubmit = useFormSubmit<FormValues>(async (values, actions) => {
+    setIsUploading(true);
     await editReleaseDataService.uploadAncillaryFile(releaseId, {
       name: values.name,
       file: values.file as File,
     });
-
+    setIsUploading(false);
     await resetPage(actions);
   }, errorCodeMappings);
 
+  const handleDelete = async (
+    ancillaryFileToDelete: string,
+    form: FormikActions<{}>,
+  ) => {
+    const ancillaryFiles = [...files];
+    const fileToDelete = ancillaryFiles.find(
+      file => file.filename === ancillaryFileToDelete,
+    );
+
+    if (!fileToDelete) {
+      return;
+    }
+    fileToDelete.isDeleting = true;
```

Comment:
Same mutation issue as mentioned in `ReleaseDataUploadsSection`

Reply:
done